docs/troubleshoot/troubleshoot-zos-certificate.md

```diff
@@ -25,12 +25,57 @@ Dkeystore.pkcs12.keyProtectionAlgorithm=PBEWithSHAAnd3KeyTripleDES -J-
 Dkeystore.pkcs12.keyPbeIterationCount=50000
 ```
 
-- Set the flag `keystore.pkcs12.legacy` enabled with no value to create a PKCS12 keystore that can be loaded.
+- Set the flag `keystore.pkcs12.legacy` to enabled with no value to create a PKCS12 keystore that can be loaded.
 
-**Notes**
+:::note
+* If you already have an existing keystore or you are using keyrings, this error will not happen.
+* If you do not use ZSS, this error will not happen because ZSS is on by default.
```

Comment:
```suggestion
* If you do not use ZSS, this error will not happen. ZSS is enabled by default.
```
Something like this maybe? the because part sounds odd to me